client/src/templates/Challenges/utils/build.ts

```diff
@@ -300,9 +300,7 @@ export function updateProjectPreview(
     const titleMatch = buildData?.sources?.index?.match(
       /<title>(.*?)<\/title>/
```

Comment:
```suggestion
      /<title(\s+(\w+=('|"|`)?.*\2)|(\w+))*>(.*?)<\/title\s*>/s
```

Reply:
Originally, I was just going to add to allow white space, but then started reading the html spec.